<ui version="4.0" >
 <class>widgeRepo</class>
 <widget class="QWidget" name="widgeRepo" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Repo</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>42</x>
     <y>160</y>
     <width>440</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title" >
    <string>Select repository</string>
   </property>
   <widget class="QRadioButton" name="radioPardus" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>40</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Pardus 2008</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioOther" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>100</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Other</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkContrib" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Contrib repo</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>130</y>
      <width>330</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="labelOther" >
       <property name="text" >
        <string>Enter Url:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineOther" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="widgetDesc" >
   <property name="geometry" >
    <rect>
     <x>42</x>
     <y>30</y>
     <width>440</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p align="center" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Please choose the repository to use while creating image&lt;/p>
&lt;p align="center" style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p align="center" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If you want to use another repository except Pardus 2008 and Contrib&lt;br /> repositories, please specify ful path of &lt;span style=" font-weight:600;">pisi-index.xml&lt;/span> file.&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
